src/index.ts

```diff
@@ -63,20 +59,20 @@ export default class TeamworkConnector extends BaseHttpConnector<
     }
 
     if (eventsToExecute.length == 0) {
-      res.send({ text: 'Error' })
+      res.send({ text: 'Error' }).status(404)
```

Comment:
Is this a 'no handler configured' error? Might want to provide a more descriptive error message?

Reply:
we're replying to the webhook call here, we should always send back a 200. Otherwise it will send the same data over and over.